report_common_hls/views/res_company_views.xml

```diff
@@ -14,6 +14,14 @@
                     <group string="Common" name="common">
                         <field name="company_chop" widget='image' class="oe_avatar"/>
                     </group>
+                    <group string="Purchase" name="purchase">
```

Comment:
We should not categorize these fields as part of 'purchase' here.  The fields should be moved under 'common'.

Reply:
I move into common group.